awswrangler/s3/_write_parquet.py

```diff
@@ -53,7 +53,7 @@ def _new_writer(
     writer: Optional[pyarrow.parquet.ParquetWriter] = None
     if not pyarrow_additional_kwargs:
         pyarrow_additional_kwargs = {}
-    if not pyarrow_additional_kwargs.get("coerce_timestamps"):
+    if "coerce_timestamps" not in pyarrow_additional_kwargs:
```

Comment:
There is another instance here:
https://github.com/aws/aws-sdk-pandas/blob/main/awswrangler/s3/_write_parquet.py#L711

Also the same thing applies for these as False would cause the same issue as None:
https://github.com/aws/aws-sdk-pandas/blob/main/awswrangler/s3/_write_parquet.py#L63

Only had a quick skim through but there might be more